src/flows/context/bucket.scoped.tsx

```diff
@@ -71,7 +71,37 @@ export const BucketProvider: FC<Props> = ({panel, children}) => {
               }
               return acc
             },
-            {system: [], user: []}
+            {
+              system: [],
+              user: [],
+              sample: [
```

Comment:
these exist inside of a query that we can use (`sample.list()`), but the result set doesn't include a human readable label for the dataset, so we're hard coding the list until it does